src/index.html

```diff
@@ -5,8 +5,297 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo" >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <section class="catalog__container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <h3 class="card__code">Product code: 195434</h3>
+          <div class="card__raiting">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
```

Comment:
If we don't have content inside we can use a single tag, but this is optional
```suggestion
              <div class="stars__star" />
```

Reply:
ok, but `Tag is not closedlinthtml(tag-close)` complains about it in VS Code